lib/Form/Label/index.js

```diff
@@ -2,9 +2,18 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import cx from 'classnames';
 
-const Label = ({ label, id, subtitle, className, highlight, active }) => {
+const Label = ({
+  label,
+  id,
+  subtitle,
+  className,
+  highlight,
+  active,
```

Comment:
I've forgotten but whats the difference between active and highlight again?

Reply:
highlight just adds the highlight style, so the little yellow border, active will give it the active style ( the yellow background)